package.json

```diff
@@ -18,14 +18,15 @@
     "format:md": "prettier --write --parser markdown '**/*.md'",
     "format:yaml": "prettier --write --parser yaml '**/*.yaml'",
     "start": "yarn playground:start",
-    "release": "npm run build; lerna publish --exact",
+    "release": "npm run build; lerna publish --exact --npm-tag next",
+    "release-to-latest": "lerna exec --no-bail --no-private --no-sort --stream -- '[ -n \"$(npm v . dist-tags.next)\" ] && npm dist-tag add ${LERNA_PACKAGE_NAME}@$(npm v . dist-tags.next) latest'",
```

Comment:
I found this here: https://github.com/lerna/lerna/issues/1364#issuecomment-379823504

Reply:
Nice.